src/main/java/model/Car.java

```diff
@@ -0,0 +1,66 @@
+package model;
+
+import model.exception.CarCreateException;
+import model.result.MoveResult;
+
+public class Car implements Comparable<Car> {
+    private static final int CAR_NAME_MAX_LENGTH = 5;
+    public static final int DEFAULT_POSITION = 0;
+
+    private String name;
+    private int position;
+
+    public Car(String name) {
+        this(name, DEFAULT_POSITION);
+    }
+
+    public Car(String name, int position) {
+        validateName(name);
+        this.name = name;
+        this.position = position;
+    }
+
+    private void validateName(String name) {
+        validateNull(name);
+        validateEmpty(name);
+        validateLength(name);
+    }
+
+    private void validateNull(String name) {
+        if (name == null) {
```

Comment:
제가 고려하지 못한 부분까지 예외처리하시고 꼼꼼하시네요 👍  
```suggestion
        if (Objects.isNull(name)) {
```
이렇게 사용하는건 어떻게 생각하시나요?
혹은 `Objects.requireNonNull(name, "차 이름에 NULL 을 입력할 수 없습니다.");` 이런 방법도 있을거 같아요.

어디까지나 개인취향이지만.. 이 부분에 대해서는 어떻게 생각하시는지 궁금해서 남겨봅니다

Reply:
이런 메소드가 있는 줄 처음 알았네요 👍
둘다 좋은데, 이번엔 requireNonNull을 사용해볼게요!